mock-core-configs/etc/mock/templates/mageia-branched.tpl

```diff
@@ -4,6 +4,7 @@ config_opts['dist'] = 'mga{{ releasever }}'  # only useful for --resultdir varia
 config_opts['extra_chroot_dirs'] = [ '/run/lock', ]
 config_opts['macros']['%distro_section'] = 'core'
 config_opts['package_manager'] = 'dnf'
+config_opts['bootstrap_forcearch'] = '{{ target_arch }}'
```

Comment:
I'm almost sure this is not correct :-( we need a precise reason why this is needed and document it.

Reply:
Can you check #1317 whether it helps?

Reply:
I did tests earlier with no success. See https://github.com/rpm-software-management/mock/issues/1304#issuecomment-1929689298.

Reply:
Ah, I see your comment in the original issue.  Lemme think about it a bit more.

Reply:
Can you please take one more look at #1317?